MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/JitterBuffer.cs

```diff
@@ -33,34 +33,59 @@ public void Lerp(RigidBodyTransform t0, RigidBodyTransform t1, float f)
 	public class Snapshot
 	{
 		/// <summary>
-		/// Trnasform identifier and respective transform.
+		/// Flags that mark special properties of the snapshot
+		/// </summary>
+		public enum SnapshotFlags : byte
```

Comment:
>public enum SnapshotFlags : byte [](start = 2, length = 32)

Should be marked as a flag with attribute.  

`[Flags]
public enum SnapshotFlags : byte`